src/coreclr/jit/importercalls.cpp

```diff
@@ -100,7 +100,8 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
 
     // Swift calls that might throw use a SwiftError* arg that requires additional IR to handle,
     // so if we're importing a Swift call, look for this type in the signature
-    CallArg* swiftErrorArg = nullptr;
+    CallArg*  swiftErrorArg = nullptr;
+    CallArgs* callArgs      = nullptr;
```

Comment:
Seems like it would be cleaner to return the `GenTree*` node from `impPopArgsForUnmanagedCall` and remove the arg in there instead.